archunit-junit/junit5/engine/src/testFixtures/java/com/another/AnotherClass.java

```diff
@@ -0,0 +1,19 @@
+package com.another;
+
+import java.io.Serializable;
+
+import com.tngtech.archunit.junit.ArchTest;
+import com.tngtech.archunit.lang.ArchRule;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
+
+public class AnotherClass {
```

Comment:
These must exist in order for `FieldSource.from` and `FieldSelector.selectField` not to throw an exception